docs/workspaces.md

```diff
@@ -364,14 +364,98 @@ write to or read from that `Workspace`. Use the `runAfter` field in your `Pipeli
 to define when a `Task` should be executed. For more information, see the [`runAfter` documentation](pipelines.md#using-the-runafter-parameter).
 
 When a `PersistentVolumeClaim` is used as volume source for a `Workspace` in a `PipelineRun`,
-an Affinity Assistant will be created. The Affinity Assistant acts as a placeholder for `TaskRun` pods
+an Affinity Assistant will be created. Available Affinity Assistant Modes are **coschedule workspace**, **coschedule pipelineruns**,
+**isolate pipelineruns** and **disabled**. 
+
+* **coschedule workspace** - The Affinity Assistant acts as a placeholder for `TaskRun` pods
 sharing the same `Workspace`. All `TaskRun` pods within the `PipelineRun` that share the `Workspace`
-will be scheduled to the same Node as the Affinity Assistant pod. This means that Affinity Assistant is incompatible
-with e.g. other affinity rules configured for the `TaskRun` pods. If the `PipelineRun` has a custom
+will be scheduled to the same Node as the Affinity Assistant pod.
+
+* **coschedule pipelineruns (WIP)** - The Affinity Assistant acts as a placeholder for the whole pipelinerun.
```

Comment:
can this be rewritten to focus less on implementation details? i.e. we really don't have to mention the affinity assistant at all; we can just say all pipelinerun pods get scheduled to the same node. (we can mention this uses inter-pod affinity even though it's an implementation detail as that may be relevant for some users due to performance concerns)

Reply:
👍 , I have rephrased the paragrah